libs/sql-parser/src/main/antlr/io/crate/sql/parser/antlr/SqlBaseParser.g4

```diff
@@ -138,7 +138,8 @@ alterStmt
     | ALTER CLUSTER DECOMMISSION node=expr                                           #alterClusterDecommissionNode
     | ALTER CLUSTER GC DANGLING ARTIFACTS                                            #alterClusterGCDanglingArtifacts
     | ALTER (USER | ROLE) name=ident
-        SET OPEN_ROUND_BRACKET genericProperties CLOSE_ROUND_BRACKET                 #alterRole
+        SET OPEN_ROUND_BRACKET genericProperties CLOSE_ROUND_BRACKET                 #alterRoleSet
+    | ALTER (USER | ROLE) name=ident RESET (property=ident | ALL)                    #alterRoleReset
```

Comment:
I chose to not allow resetting multiple settings at once, following PostgreSQL behavior, but we can easily support this and allow a comma separated list of settings (same as in SET).